src/components/layouts/ScrollToTop.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+
+import ArrowUp from '../../assets/icons/icons-arrow-up.png';
+
+const ScrollToTop = () => {
+  const [isVisible, setIsVisible] = useState(false);
+
+  const toggleVisibility = () => {
+    if (window.scrollY > 50) {
+      setIsVisible(true);
+    } else {
+      setIsVisible(false);
+    }
+  };
+
+  const scrollToTop = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: 'smooth',
+    });
+  };
+
+  useEffect(() => {
+    window.addEventListener('scroll', toggleVisibility);
+    return () => {
+      window.removeEventListener('scroll', toggleVisibility);
+    };
+  }, []);
+
+  return (
+    <>
+      {isVisible && (
+        <button
```

Comment:
`alt="Arrow Up" works for the img but we can make this button even more accessible with an aria-label for the button. Something like "Scroll to top" will enhance accessibility.